src/bin/buildRange.ts

```diff
@@ -66,6 +66,14 @@ const REFERENCE_RELEASES: ReleaseDefinition[] = [
     startVersion: 87,
     iterationsPattern: [2, 2],
   },
+  {
+    startVersion: 89,
```

Comment:
This is for nightly, so this should be 90 right?

Reply:
oh, yep.